### ISSUE TITLE
python3: pip3 and easy_install now completely missing?

### ISSUE THREAD

**Issue by w-och** [mistydemeo/tigerbrew#599]
Hi misty, this new version of python3 that you added seems to completely remove the pip3 executable and easy_install for python3 is also missing. Neither seems to be available in the location where the installer suggests they may be.

**Comment by w-och** [comment]
==> Downloading https://files.pythonhosted.org/packages/ae/e8/2340d46ecadb1692a1e455f13f75e596d4eab3d11a57446f08259dee8f02/pip-10.0.1.tar.gz
Already downloaded: /Users/w/Library/Caches/Homebrew/python3--pip-10.0.1.tar.gz
==> Downloading https://files.pythonhosted.org/packages/2a/fb/aefe5d5dbc3f4fe1e815bcdb05cbaab19744d201bbc9b59cfa06ec7fc789/wheel-0.31.1.tar.gz
Already downloaded: /Users/w/Library/Caches/Homebrew/python3--wheel-0.31.1.tar.gz
==> /usr/local/Cellar/python3/3.7.0/bin/python3 -s setup.py --no-user-cfg install --force --verbose --install-scripts=/usr/local/Cellar/python3/3.7.0/bin --install-lib=/usr/local/lib/python3.7/site-packages -
Last 15 lines from /Users/w/Library/Logs/Homebrew/python3/01.python3:

Traceback (most recent call last):
  File "setup.py", line 11, in <module>
    import setuptools
  File "/usr/local/Cellar/python3/3.7.0/libexec/setuptools/setuptools/__init__.py", line 12, in <module>
    import setuptools.version
  File "/usr/local/Cellar/python3/3.7.0/libexec/setuptools/setuptools/version.py", line 1, in <module>
    import pkg_resources
  File "/usr/local/Cellar/python3/3.7.0/libexec/setuptools/pkg_resources/__init__.py", line 35, in <module>
    import plistlib
  File "/usr/local/Cellar/python3/3.7.0/Frameworks/Python.framework/Versions/3.7/lib/python3.7/plistlib.py", line 65, in <module>
    from xml.parsers.expat import ParserCreate
  File "/usr/local/Cellar/python3/3.7.0/Frameworks/Python.framework/Versions/3.7/lib/python3.7/xml/parsers/expat.py", line 4, in <module>
    from pyexpat import *
ModuleNotFoundError: No module named 'pyexpat'
Warning: The post-install step did not complete successfully
You can try again using `brew postinstall python3`
==> Caveats
Python has been installed as
  /usr/local/bin/python3

Unversioned symlinks `python`, `python-config`, `pip` etc. pointing to
`python3`, `python3-config`, `pip3` etc., respectively, have been installed into
  /usr/local/opt/python3/libexec/bin

**Comment by w-och** [comment]
% ls -la /usr/local/opt/python3/libexec/bin

total 32
drwxr-xr-x  6 w  admin  204 Jul 16 15:09 .
drwxr-xr-x  6 w  admin  204 Jul 16 15:09 ..
lrwxr-xr-x  1 w  admin   58 Jul 16 15:09 idle -> ../../Frameworks/Python.framework/Versions/3.7/bin/idle3.7
lrwxr-xr-x  1 w  admin   59 Jul 16 15:09 pydoc -> ../../Frameworks/Python.framework/Versions/3.7/bin/pydoc3.7
lrwxr-xr-x  1 w  admin   60 Jul 16 15:09 python -> ../../Frameworks/Python.framework/Versions/3.7/bin/python3.7
lrwxr-xr-x  1 w  admin   68 Jul 16 15:09 python-config -> ../../Frameworks/Python.framework/Versions/3.7/bin/python3.7m-config

**Comment by w-och** [comment]
% ls -la /usr/local/bin/pip*

zsh: no matches found: pip*

**Comment by w-och** [comment]
I am on 10.5/PPC

**Comment by w-och** [comment]
https://gist.github.com/w-och/ea6d2e6adfeaad482d74e282e445dfe5

**Comment by mistydemeo** [comment]
Hmm, I wonder if the system expat is too old on 10.5. Do you happen to have the full build logs, including the `make`?

**Comment by w-och** [comment]
Here you go. http://s000.tinyupload.com/?file_id=09436027085599333607

**Comment by mistydemeo** [comment]
Looks like pyexpat failed because of a compiler flag it was passing:

```
building 'pyexpat' extension
creating build/temp.macosx-10.5-ppc-3.7/private/tmp/python320180716-60230-1e16ojj/Python-3.7.0/Modules/expat
gcc-4.2 -fno-strict-aliasing -Wsign-compare -fno-common -dynamic -DNDEBUG -g -O3 -Wall -std=c99 -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Werror=implicit-function-declaration -DHAVE_EXPAT_CONFIG_H=1 -DXML_POOR_ENTROPY=1 -DUSE_PYEXPAT_CAPI -I/private/tmp/python320180716-60230-1e16ojj/Python-3.7.0/Modules/expat -I./Include -I. -I/usr/local/include -I/private/tmp/python320180716-60230-1e16ojj/Python-3.7.0/Include -I/private/tmp/python320180716-60230-1e16ojj/Python-3.7.0 -c /private/tmp/python320180716-60230-1e16ojj/Python-3.7.0/Modules/pyexpat.c -o build/temp.macosx-10.5-ppc-3.7/private/tmp/python320180716-60230-1e16ojj/Python-3.7.0/Modules/pyexpat.o -Wno-implicit-fallthrough
cc1: error: unrecognized command line option "-Wno-implicit-fallthrough"
```

I'll look at removing that flag.

**Comment by w-och** [comment]
Thanks a lot!

**Comment by w-och** [comment]
By the way shouldn't this be using gcc-7 which is installed?

**Comment by w-och** [comment]
I just used --cc="gcc-7" and it fixed it misty.

**Comment by mistydemeo** [comment]
This should be fixed with GCC 4.2 and 4.0 now. Thanks for the report!
